README.rst

```diff
@@ -54,7 +54,7 @@ within the CMake source directory or any other build directory of your
 choice. Once this has finished successfully, run ``make`` and
 ``make install``.  In summary::
 
- $ ./bootstrap && make && make install
+ $ ./bootstrap && gmake && gmake install
```

Comment:
@Kentzo 
1. I built CMake on next Linux system:

`gaal@linux-t420:~> uname -a
Linux linux-t420 4.1.26-21-default #1 SMP PREEMPT Mon Jun 13 13:32:30 UTC 2016 (294632f) x86_64 x86_64 x86_64 GNU/Linux
gaal@linux-t420:~> cat /etc/os-release 
NAME="openSUSE Leap"
VERSION="42.1"
VERSION_ID="42.1"
PRETTY_NAME="openSUSE Leap 42.1 (x86_64)"
ID=opensuse
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:opensuse:42.1"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://opensuse.org/"
ID_LIKE="suse"`

2. I used `gmake` because it was told to me by `bootstrap` utility clearly